online_meetings/requests_zoom_jwt.py

```diff
@@ -0,0 +1,78 @@
+"""
+    Class for managing tokens, expirations and generations.
+    Currently just handles JWT for Zoom
+"""
+from datetime import datetime, timedelta
+import requests
+from requests.auth import AuthBase
+import jwt
+
+
+class ZoomJWT(AuthBase):
+
+    @property
+    def token(self) -> str:
+        """Check if the token is expired, if so generate a new token and return the token either way.
+        
+        :return: Returning the token value as a string
+        :rtype: str
+        """
+        # If token is close to expiring just re-create
+        if datetime.utcnow() > self.expiration - timedelta(seconds=30):
```

Comment:
Nitpick: Might want to wrap `self.expiration - ...` in parentheses, just for clarity.

Reply:
Also, might not hurt to have a debug logging message here for visibility on the token re-creation.

Reply:
Thanks I wanted to add a little more debug to here, I'll add that.